dfastmi/io/Reach.py

```diff
@@ -0,0 +1,39 @@
+from abc import ABC
```

Comment:
don't forget documentation.

Reply:
will add

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112

Reply:
will be done in reach refactoring issue